modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -41,7 +41,11 @@ public int doStartTag() throws JspException {
 	}
 
 	public String getClearAllTitle() {
-		return _clearAllTitle;
+		if (Validator.isNotNull(_clearAllTitle)) {
```

Comment:
This default declaration is probably better to be handled directly in the property definition. We only do it in the getters when a model in involved and we can't handle it in other way.

Reply:
Sure undestood, I was just following [this](https://github.com/liferay-frontend/liferay-portal/pull/528#discussion_r542400881) advice. We've changed that but I'll let @kresimir-coko add a link to the comimit.

Reply:
Hey @carloslancha fixed this in [10aa8d6](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/10aa8d6a93f30cf9cf83cd9c7e846dddd32c2f24).

A couple important notes:
- SF didn't succeed with `"Clear All"`, but was successful with `"clear-all"`
- The `closeButtonAriaLabel` prop has a default value of `"Close {0}"`
  - After testing it, I found out that it's specific to a `selectedItem` and thus has different value depending on `selectedItem.label`

Reply:
> @carloslancha
> 
> > @kresimir-coko @julien is this ready for review?
> 
> Yep! I've pushed 3 commits, and linked them in comments which they solved.

Hey @kresimir-coko let's not confuse @carloslancha :joy:
I thought we agree'd you'd look at [this](https://github.com/liferay/clay/issues/3850) issue in Clay?

Reply:
We did, I tested it and reported feedback [here](https://github.com/liferay-frontend/liferay-portal/pull/528#discussion_r543403915). And now I'm making a PR on Clay.

Reply:
>SF didn't succeed with "Clear All", but was successful with "clear-all"

Yes, to localize strings you need to follow that key pattern.

>After testing it, I found out that it's specific to a selectedItem and thus has different value depending on selectedItem.label

Not sure what you mean. ClayMultiSelect receives `closeButtonAriaLabel` param, that defaults to `Remove {0}`, where `{0}` will be replaced with `selectedItem[locator.label]` (you can find that [here](https://github.com/liferay/clay/blob/6b00a1e60deaa3c415dbed5227c3ca6017ac14ea/packages/clay-multi-select/src/index.tsx#L291).

So we have to do the same, default value needs to be the same as in clay (`Remove {0}`), and where we render it in java use the utility to localize passing variables (`LanguageUtil.format` I guess).

The tricky part here is that you need the key to be in key format `remove-x`. So:

- `closeButtonAriaLabel` default value should be `remove-x` 
- `remove-x` language property needs to have the format `remove-x=Remove {0}`
- `closeButtonAriaLabel` prop passed to the js component should be `LanguageUtil.get(TagResourceBundleUtil.getResourceBundle(pageContext), "remove-x"));`
- Where we're rendering this in server side we need to localize it passing variables (using `LanguageUtil.format`)

Reply:
@carloslancha while we wait for the ClayButtonWithIcon change to get merged (I assume we need that merged before this, or maybe not?) I'll work on this `remove-x` thingy.

Reply:
There's a possible problem with this one @carloslancha, atleast server side related.

We're using `LabelTag labelTag = new LabelTag();` to render these labels that have the `dismissable` property, and that dismissable button gets the aria label.

The problem is that the `LabelTag` doesn't have a setter for that aria label.

- Do I add the setter as part of this, already, big PR?
- Do we wait for this to get merged to worry about the aria label?
- Do I open a new ticket/PR for adding of the setter and wait for that to get merged before we can finish MultiSelect?

Reply:
Since aria-label in server side doesn't affect to the visual markup at all we can just ignore it and only worry about what is passed to the js component.

Reply:
👍

Reply:
@carloslancha I've pushed a couple commits related to this, namely [a3c0c48](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/a3c0c48c8dd4c51fb60ad930ba52ab2b8d2275c8).

I couldn't get it to work, so if you or @julien can spare some time tomorrow and maybe help me out. I'll try it again solo, maybe some new ideas come up, but I've exhausted everything I had today trying to get it to work.

The problem is that the value supplied in the JSP for that prop doesn't get properly passed to JS, so that can be investigated further, and also, the default value of `remove-x` doesn't get translated to `Remove {0}` once it gets to JS.

Reply:
Taking a look...